src/Components/Suggestions/Suggestions.jsx

```diff
@@ -1,11 +1,12 @@
 import style from "./Suggestions.module.css";
+import { SecondaryHeading } from "../SecondaryHeading/SecondaryHeading";
 
-export const Suggestions = ({ updateFormData }) => {
+export const Suggestions = ({ updateFormData, value }) => {
   const thirdQuestion = (e) => updateFormData("thirdQuestion", e.target.value);
 
   return (
     <div>
-      <h2>3. What suggestions do you have to improve your shopping experience?</h2>
+      <SecondaryHeading question="3. What suggestions do you have to improve your shopping experience?" />
```

Comment:
@mikaeloling  I changed the h2 heading to a Component that I created for headings. Because it repeats multiple times!!